packages/pwa/package.json

```diff
@@ -184,11 +184,26 @@
     ],
     "l10n": {
         "supportedLocales": [
-            "en-GB",
-            "fr-FR",
-            "it-IT",
-            "zh-CN",
-            "ja-JP"
+            {
+                "id": "en-GB",
+                "currencyCode": "GBP"
+            },
```

Comment:
Do we need to update [this readme](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/17405cf6b86b6581e9c9c7ae61534b8e973c57ca/packages/pwa/app/translations/README.md) too?